src/asynchronous/client.rs

```diff
@@ -45,6 +46,7 @@ impl Client {
     pub fn new(fd: RawFd) -> Client {
         let stream = utils::new_unix_stream_from_raw_fd(fd);
```

Comment:
The fd will be closed when ClientClose dropping

Reply:
hi, It looks like async client didn't hold ClientClose? If I miss?